shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/model/Token.kt

```diff
@@ -0,0 +1,10 @@
+package co.nimblehq.ic.kmm.suv.domain.model
+
+data class Token(
+    val accessToken: String,
+    val tokenType: String,
+    val expiresIn: Int,
+    val refreshToken: String,
+    val createdAt: Int
+)
+
```

Comment:
extra new line?

Reply:
Fixed in [`fa34633` (#87)](https://github.com/suho/kmm-ic/pull/87/commits/fa346335edd4ca03014c9ddd5b53f6bd85f9e63a)